<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>singleEventDialog</class>
 <widget class="QDialog" name="singleEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_subject">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>主旨</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>59</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>時間</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_reminder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>195</y>
     <width>59</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提醒</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_location">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>59</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地點</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_note">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>59</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>備註</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subject">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="time_year">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="time_month">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="time_date">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_year">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>45</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_month">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>45</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>45</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_duration">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>持續時間</string>
   </property>
  </widget>
  <widget class="QListWidget" name="duration">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1 hr </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 hr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 hr</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="location">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="reminder">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>當事件發生時</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10 min before</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30 min before</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1 hr before</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="note">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>320</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_start_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>開始時間</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_time">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
